custom_components/abbfreeathome_ci/manifest.json

```diff
@@ -8,9 +8,9 @@
   "homekit": {},
   "iot_class": "local_push",
   "loggers": ["abbfreeathome"],
-  "requirements": ["local-abbfreeathome==2.1.2"],
+  "requirements": ["local-abbfreeathome==2.2.1"],
   "ssdp": [],
-  "version": "1.4.4",
+  "version": "1.5.0",
```

Comment:
Let's hold off on bumping the version, you've got 2 PR's. I'll merge in a final change to bump the version once the two PR's are merged in.